zebra-state/src/constants.rs

```diff
@@ -18,7 +18,7 @@ pub use zebra_chain::transparent::MIN_TRANSPARENT_COINBASE_MATURITY;
 pub const MAX_BLOCK_REORG_HEIGHT: u32 = MIN_TRANSPARENT_COINBASE_MATURITY - 1;
 
 /// The database format version, incremented each time the database format changes.
-pub const DATABASE_FORMAT_VERSION: u32 = 14;
+pub const DATABASE_FORMAT_VERSION: u32 = 15;
```

Comment:
The current version is 22, I'm not expecting any database format changes in the next week:
```suggestion
pub const DATABASE_FORMAT_VERSION: u32 = 23;
```